packages/atomic/src/components/search/result-template-components/atomic-quickview-modal/atomic-quickview-modal.tsx

```diff
@@ -82,6 +82,10 @@ export class AtomicQuickviewModal implements InitializableComponent {
 
   private interactiveResult?: InteractiveResult;
 
+  public componentWillRender(): void {
+    this.minimizeSidebar = this.bindings.store.isMobile();
```

Comment:
This behaviour should be documented, in JSdoc, on the property

Reply:
Does your comment still apply if `minimizeSidebar` is a state ?